system-tests/e2e-suite/src/test/scala/com/hotels/styx/client/ExpiringConnectionSpec.scala

```diff
@@ -36,31 +41,38 @@ import scala.concurrent.duration._
 
 class ExpiringConnectionSpec extends FunSpec
   with StyxProxySpec
-  with StyxClientSupplier
   with Eventually {
 
   val mockServer = FakeHttpServer.HttpStartupConfig()
     .start()
     .stub(urlStartingWith("/app1"), aResponse
       .withStatus(200)
+      .withHeader("ExpiringConnectionSpec", "x")
       .withHeader(TRANSFER_ENCODING, CHUNKED)
       .withBody("I should be here!")
     )
 
-  override val styxConfig = StyxConfig()
+  val styxConfig = StyxConfig()
+
+  var pooledClient: StyxHttpClient = _
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
+    val request = get(s"http://localhost:${mockServer.port()}/app1/origin-ping").build()
+    val resp = decodedRequest(request)
+    resp.status() should be(OK)
+    resp.bodyAs(UTF_8) should be("I should be here!")
```

Comment:
The above block is not strictly necessary for this test. Consider removing it, or alternatively wrap it inside a function called `verifyStyxIsAlive` etc.